spec/panoptes/client/subjects_spec.rb

```diff
@@ -15,21 +17,21 @@
     let(:client) { application_client }
 
     it 'returns true if a subject is accessible to a project' do
-      result = client.subject(72850, project_id: 1315)
+      result = client.subject(72_850, project_id: 1315)
```

Comment:
Is this a rubocop thing? Seems kind of weird.

Reply:
yep - happy to turn this off, https://github.com/rubocop-hq/ruby-style-guide#underscores-in-numerics

Reply:
Cool, that's the totality of my opinion. Hard to argue that it's not easier to read, up to you.